content/docs/installation/orchestrator/orchestrator-helm-repository.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Deploy from repository
+date: "2024-03-12"
+---
+
+{{< remoteMD "https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/README.md?raw=true" >}}
```

Comment:
Not sure if related to this PR, but once rendered this page has a link to workflow installation
![image](https://github.com/parodos-dev/parodos-dev.github.io/assets/15085664/9cdc8c51-410f-4693-9f53-1d58b3bcef9f)

If i click this I'm redirected to https://www.parodos.dev/serverless-workflows-helm/ and i get: 
![image](https://github.com/parodos-dev/parodos-dev.github.io/assets/15085664/ae9d503b-af3c-4a5e-9ad5-7e7fda24fe88)
while if i click from the link available in the installation page, I'm redirect to https://www.parodos.dev/docs/installation/serverless-workflows-helm/ and I get: 
![image](https://github.com/parodos-dev/parodos-dev.github.io/assets/15085664/9d33cdc7-01e5-4fa3-b938-424921e8d1f5)

Is it ok?

Reply:
IMO it's not related to this PR, it's just that our doc is starting to be a mess and we need to harmonize/clean it in all the places we have docs (so the gh-pages of the helm chart repos and this repo for instance)

Reply:
sadly yes, this page needs to be fixed on the origin repository. but the link is correct.

Reply:
> sadly yes, this page needs to be fixed on the origin repository. but the link is correct.

I'll correct my answer and point directly to the gh-pages which contains the update documentation:
https://github.com/parodos-dev/serverless-workflows-helm/blob/gh-pages/README.md